autoload/go/oracle.vim

```diff
@@ -56,7 +56,7 @@ func! s:qflistSecond(output)
     " We discard line2 and col2 for the first errorformat, because it's not
     " useful and quickfix only has the ability to show one line and column
     " number
-	let &errorformat = "%f:%l.%c-%.%#:\ %m,%f:%l:%c:\ %m"
+	let &errorformat = "%f:%l.%c-%[%^:]%#:\ %m,%f:%l:%c:\ %m"
```

Comment:
Got it, makes sense now. I wasn't sure about messages containing `:`, so it was a fix but seems it has some quirks in it. Thanks for the contribution btw, much appreciated.